lib/js_routes.rb

```diff
@@ -19,7 +19,8 @@ class JsRoutes
     url_links: nil,
     camel_case: false,
     default_url_options: {},
-    compact: false
+    compact: false,
+    serializer: "this.default_serializer"
```

Comment:
The way we can make it clear is to make `default_serializer` a public method and let default be `JsRoutes.default_serialize`. In this way people can clearly see where is the default and even test in console it in console comparing to other serializers.